### PR TITLE
AppAuth release 1.3.1

### PR DESCRIPTION
**Pull request by nonik0**
Bug fixes and cleanup, bumping NuGet version

### CODE REVIEW

File: sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/Helpers/EnvironmentHelper.cs [hunk 1]
```diff
@@ -31,6 +31,14 @@ public static string SystemDirectory
                 return sb.ToString();
             }
         }
+#else
+        public static string SystemDirectory
```
Comment (AlexGhiondea) [inline sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/Helpers/EnvironmentHelper.cs]: Suggestion: #if/def inside the property getter -- that way it is less likely that the 2 will diverge in the future.

Comment (nonik0) [reply sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/Helpers/EnvironmentHelper.cs]: Good suggestion, done.

**Review by AlexGhiondea — APPROVED** [review]

**Pull request merged**